README.md

```diff
@@ -82,58 +82,29 @@ sgx-lkl-setup
 B. Building SGX-LKL-OE from source
 ----------------------------------
 
-SGX-LKL has been tested on Ubuntu Linux 18.04.
-
-### 1. Install the SGX-LKL branch of the Open Enclave SDK 
-
-To run SGX-LKL in SGX enclaves, it relies on an installation of the *Open Enclave 
-SDK* (https://openenclave.io/sdk/). The current version of SGX-LKL-OE requires a
-modified version of the Open Enclave SDK, which can be found here: 
-https://github.com/openenclave/openenclave (branch: feature/sgx-lkl-support).
-
-1. Clone the `feature.sgx-lkl` branch of the Open Enclave SDK:
-```
-git clone -b feature/sgx-lkl-support git@github.com:openenclave/openenclave.git openenclave-sgxlkl
-```
-
-2. Install the Open Enclave build requirements:
-```
-cd openenclave-sgxlkl
-sudo scripts/ansible/install-ansible.sh
-sudo ansible-playbook scripts/ansible/oe-contributors-setup.yml
-```
-
-3. Build the Open Enclave SDK:
-```
-mkdir build
-cd build
-cmake -G "Unix Makefiles" -DCOMPILE_SYSTEM_EDL=ON ..
-make
-sudo make install
-```
-
-4. Source the Open Enclave SDK configuration script:
-```
-source /opt/openenclave/share/openenclave/openenclaverc
-```
-
-### 2. Building SGX-LKL as part of the source tree
+SGX-LKL has been tested on Ubuntu Linux 18.04 and with a gcc compiler
+version of 7.4 or above. Older compiler versions may lead to compilation
+and/or linking errors.
 
 1. Install the SGX-LKL build dependencies:
 ```
 sudo apt-get install make gcc g++ bc python xutils-dev bison flex libgcrypt20-dev libjson-c-dev automake autopoint autoconf pkgconf libtool libcurl4-openssl-dev libprotobuf-dev libprotobuf-c-dev protobuf-compiler protobuf-c-compiler libssl-dev
 ```
 
-Compilation has been tested with versions 7.4 of gcc. Older compiler versions
-may lead to compilation and/or linking errors.
-
 2. Clone the SGX-LKL git repository:
 ```
-git clone --branch oe_port git@github.com:lsds/sgx-lkl.git sgx-lkl
+git clone --branch oe_port git@github.com:lsds/sgx-lkl.git
 cd sgx-lkl
 ```
 
-3. Build SGX-LKL in the source tree:
+3. Install the Open Enclave build dependencies
+```
+cd openenclave
+sudo scripts/ansible/install-ansible.sh
+sudo ansible-playbook scripts/ansible/oe-contributors-setup.yml
```

Comment:
Be careful here, this installs the driver as well. We may want to offer multiple options depending on the environment.

Reply:
Ok, this is just copied from what we currently have in the README file. What are the different scripts that we should mention here? I can then update the text as part of this PR.

Reply:
If you run on Azure, then scripts/ansible/oe-contributors-acc-setup-no-driver.yml since new ACC VMs already contain the driver. If you run outside Azure, then likely scripts/ansible/oe-contributors-setup.yml which unfortunately also installs the driver and they don't have a script without the driver.

Reply:
Done